addons/website_forum/i18n/website_forum.pot

```diff
@@ -3118,6 +3118,13 @@ msgstr ""
 msgid "Sorry, we could not find any <b>%s</b> result <b>%s</b> %s%s%s."
 msgstr ""
 
+#. module: website_forum
+#. odoo-javascript
```

Comment:
we are in master now, you don't need to change the pot file

Reply:
Hello @std-odoo 

made the changes, please have a look at it.